src/coreclr/jit/emitarm64.cpp

```diff
@@ -14970,6 +14988,7 @@ void emitter::emitIns_Call(EmitCallType          callType,
         case INS_sve_ld2w:
         case INS_sve_ld2d:
         case INS_sve_ld2q:
+        case INS_sve_splice: // SVE_CV_3A
```

Comment:
looking at this method, not sure why there is inconsistency regarding comments. Are we commenting only the instructions that has multiple formats?